dotnetapp-dev/README.md

```diff
@@ -62,21 +62,21 @@ cd dotnetapp-dev
 docker build --target testrunner -t dotnetapp-dev:test .
 ```
 
-The following commands rely on [volume mounting](https://docs.docker.com/engine/admin/volumes/volumes/) to enable the test runner to write test log files to your local drive. Without that, running tests as part of `docker run` isn't as useful. You need to [enable shared drives](https://docs.docker.com/docker-for-windows/#shared-drives) or [file sharing](https://docs.docker.com/docker-for-mac/#file-sharing) for this to work, depending on your operating system. It is straightforward, so please do give it a try.
+The following commands rely on [volume mounting](https://docs.docker.com/engine/admin/volumes/volumes/) (that's the `-v` argument in the following commands) to enable the test runner to write test log files to your local drive. Without that, running tests as part of `docker run` isn't as useful.
 
-You can run the sample on **Windows** using Windows containers using the following command.
+You can run the sample on **Windows** using Windows containers using the following command. You should [enable shared drives](https://docs.docker.com/docker-for-windows/#shared-drives) first.
```

Comment:
The shared-drives reference isn't applicable to Windows containers.  The concept only applies to Linux containers on Windows hosts.

Reply:
OK. I wasn't quite clear on that. I switched between the two modes on windows and was confused. I don't see that UI in either case. I'll switch it.

Reply:
If you switch to Linux mode, go to the docker settings window, the second tab on the left is 'Shared Drives'